scripts/qtsupport.py

```diff
@@ -507,20 +508,109 @@ def filter(self, p, columns):
             item.setHidden(all([unicode(item.text(column)).lower().find(p) == -1
                                 for column in columns]))
 
-""" TODO implement this option
-class SchStaticPage(QWizardPage):
-    def __init__(self, parent):
-        super(SchStaticPage, self).__init__(parent)
-        self.setTitle("Manually create a schedule entry")
+# TODO implement this option
```

Comment:
Why change this in this PR?

Reply:
Mentioned in initial comment when opening PR. It messes up syntax highlighting in vim sometimes, makes no difference otherwise. Can be removed if you insist.

Reply:
To be clear: you're saying that using `"""` style comment sections is a problem for formatting sometimes? You think that should always be avoided outside of doc strings, then? Not unreasonable, if so, just I wasn't aware.

Reply:
Yes, it looks so. Can't say with 100% certainty, vim syntax highlighting acts strange sometimes. Also `"""` is meant for doc strings, `#` is a proper way to comment out code blocks, I think.